packages/intent/src/discovery/walk.ts

```
@@ -0,0 +1,113 @@
+import { existsSync, readFileSync } from 'node:fs'
+import { join } from 'node:path'
+import { listNodeModulesPackageDirs, resolveDepDir, getDeps } from '../utils.js'
```

ESLint

[error] Member 'getDeps' of the import declaration should be sorted alphabetically.

(sort-imports)